### PR TITLE
fix: Build docs without parallelization to fix pop-up issues with sphinx-tippy

### PR DESCRIPTION
**Pull request by nfelt14**
## Proposed changes

Run doc building without parallelization to try to work around a bug in sphinx-tippy.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply and remove the ones that don't_

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Documentation update (an update to enhance the user experience when reading through the docs)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have followed the guidelines in the [CONTRIBUTING](https://github.com/tektronix/tm_devices/blob/main/CONTRIBUTING.md) document
- [x] I have signed the CLA
- [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/tektronix/tm_devices/pulls) for the same update/change
- [x] I have performed a self-review of my code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Basic linting passes locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added necessary documentation (if appropriate)
- [x] I have updated the Changelog with a brief description of my changes

### CODE REVIEW

**Comment by nfelt14** [comment]
A normal doc build used to take about 45 minutes, but the pop-ups were non-functional. Now the build takes 65 minutes, but the pop-ups are functional.

**Comment by nfelt14** [comment]
This PR works around this issue: https://github.com/sphinx-extensions2/sphinx-tippy/issues/9

File: .github/workflows/test-docs.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ concurrency:
 jobs:
   # Test building the docs (html, linkcheck, doctest, coverage)
   test-docs:
-    runs-on: ubuntu-latest-4-cores
+    runs-on: macos-latest
```
Comment (nfelt14) [inline .github/workflows/test-docs.yml:16]: I think this change is going to need to be reverted. It is causing the build to slow down significantly.

Comment (nfelt14) [reply .github/workflows/test-docs.yml]: I reverted this change.

**Review by michaelwagoner — APPROVED** [review]

**Review by michaelwagoner — APPROVED** [review]

**Pull request merged**